deploy/charts/cert-manager/templates/podmonitor.yaml

```diff
@@ -45,6 +45,6 @@ spec:
       scrapeTimeout: {{ .Values.prometheus.podmonitor.scrapeTimeout }}
       honorLabels: {{ .Values.prometheus.podmonitor.honorLabels }}
       {{- with .Values.prometheus.podmonitor.endpointAdditionalProperties }}
-      {{- toYaml . | nindent 4 }}
+      {{- toYaml . | nindent 6 }}
```

Comment:
Good catch.
We should create an "all options enabled" values.yaml file and apply some yaml linting on the result.
Helm tests could be another solution for this problem long-term.